examples/lighting-app/qpg/src/AppTask.cpp

```diff
@@ -68,6 +70,8 @@ bool sIsThreadProvisioned = false;
 bool sIsThreadEnabled     = false;
 bool sHaveBLEConnections  = false;
 
+uint8_t sTestEventTriggerEnableKey[TestEventTriggerDelegate::kEnableKeyLength] = { 0x00, 0x11, 0x22, 0x33, 0x44, 0x55, 0x66, 0x77,
```

Comment:
You must make sure this is not enabled for field devices if people cut and paste your example. Otherwise a commissioner may detect TestEventTriggersEnabled attribute true and deny commissioning in production